doc/esmvalcore/recipe.rst

```diff
@@ -47,27 +58,33 @@ recipe_ocean_amoc.yml.
       projects:
         - ukesm
 
-Note that the authors, projects, and references will need to be included in the
-``config-references.yml`` file. The author name uses the format:
-`surname_name`. For instance, Mickey Mouse would be: `mouse_mickey`.
-Also note that this username is unlikely to be the same as the github
-user name.
+.. note::
 
+   Note that all authors, projects, and references mentioned in the description
+   section of the recipe need to be included in the ``config-references.yml``
+   file. The author name uses the format: ``surname_name``. For instance, John
+   Doe would be: ``doe_john``. This information can be omitted by new users
```

Comment:
How do you omit the author information? You just don't put that information in the recipe?

Reply:
Yes